src/idvc/ui/save_widgets.py

```diff
@@ -0,0 +1,106 @@
+from PySide2 import QtWidgets, QtCore
+from PySide2.QtWidgets import *
+from PySide2.QtCore import *
+from PySide2.QtGui import *
+import shutil
+import os
+import tempfile
+from eqt.threading import Worker
+
+
+class SaveObjectWindow(QtWidgets.QWidget):
```

Comment:
Is this not a Dialog? And why not one from EQT?

Reply:
You are right but I did not touch this class at all, I only separated it from the graphs class! I tried to edit it a bit and I destroyed it so I went back to what it was originally.